openapi/resource-server.yaml

```diff
@@ -1324,6 +1424,27 @@ components:
           kty: OKP
           crv: Ed25519
           x: oy0L_vTygNE4IogRyn_F5GmHXdqYVjIXkWs2jky7zsI
+    feeStructure:
+      title: feeStructure
+      type: object
+      properties:
+        fixed:
+          $ref: ./schemas.yaml#/components/schemas/amount
+        percentage:
```

Comment:
should we specify what percentage is, ie. whether 1% should be specified as 0.01 vs 1?
Alternatively, we can just specify an additional example that includes the feeStructure

Reply:
I talked to @BlairCurrey and we were thinking about using basis points and integers instead. What do you think?